assets_src/config/gameplay/units/villager.yaml

```diff
@@ -5,7 +5,6 @@ modifyingUnitClasses:
     replace:
         isRecruitable: true
         tags:
-            - villager
```

Comment:
why remove villager? This was a default tag, and I'm not sure what is was used for.

Reply:
I was experimenting, I can add it back in